docs/deployment/under_the_hood.md

```diff
@@ -62,7 +62,7 @@ The alternative serve methods `'direct'` and `'redirect'` work by serving the do
 
 If a remote ("cloud") storage backend is used, the serve method will default to `'redirect'` and the document will be served directly from the cloud storage file url. In this case, users may be able to bypass permission checks, and scripts within documents may be executed (depending on the cloud storage service's configuration). However, the impact of cross-site scripting attacks is reduced, as the document is served from a different domain to the main site.
 
-If these limitations are not acceptable, you may set the `WAGTAILDOCS_SERVE_METHOD` to `serve_view` and ensure that the documents are not publicly accessible using the cloud service's file url.
+If these limitations are not acceptable, you may set the `WAGTAILDOCS_SERVE_METHOD` to `serve_view` and ensure that the documents are not publicly accessible using the cloud service's file url. A Content Security Policy is also automatically set for you.
```

Comment:
Putting this sentence here feels like a bit of a non-sequitur - this paragraph is about what you should do if you consider the default setup for remote docs to be not locked-down enough.

I'd suggest instead adding it to the bullet point that ends with "this ensures that if the document is a type that can contain scripts (such as HTML or SVG), the browser is prevented from executing them."

Reply:
With the configuration setting, I've tweaked the wording further.